src/components/Stepper.vue

```diff
@@ -6,80 +6,80 @@
             class="bg-secondary text-primary"
             flat
             header-nav
-            inactive-icon="star"
+            inactive-icon=""
             ref="stepper"
-            v-bind:model-value="step.step"
+            v-bind:model-value="step"
             v-on:update:modelValue="updateStep"
             vertical
         >
             <q-step
                 color="primary"
-                icon="star"
+                icon=""
                 title="Start"
-                v-bind:done="step.step.value > 1"
+                v-bind:done="false"
```

Comment:
Can you please remove all `v-bind:done` properties?

Reply:
done